core/organization/service.go

```diff
@@ -184,6 +184,11 @@ func (s Service) AddMember(ctx context.Context, orgID, relationName string, prin
 	}); err != nil {
 		return err
 	}
+
+	audit.GetAuditor(ctx, orgID).Log(audit.OrgMemberCreatedEvent, audit.Target{
```

Comment:
Do we have events for whenever a user is added via SSO login? (Or any other way of adding a user)
We need to emit an event for all those scenarios.

Reply:
All the usecases should be calling this method so should be covered.